modules/seo-tools/jetpack-seo.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * An SEO expert walks into a bar, bars, pub, public house, Irish pub, drinks, beer, wine, liquor, Grey Goose, Cristal...
+ */
+class Jetpack_SEO {
+	public function __construct() {
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	public function init() {
+		if ( apply_filters( 'jetpack_seo_meta_tags', true ) ) {
+			add_action( 'wp_head', array( $this, 'meta_tags' ) );
```

Comment:
I've added a second filter before outputting non-OG SEO meta tags in 8449dcbf6efad26e506ddf762de9d2a212cd3617. There is also a short-circuit filter that can be used to skip meta tags processing there.